frontend/src/components/single-contract/SingleContractData.tsx

```diff
@@ -7,10 +7,12 @@ interface ISingleContractDataProps {
 }
 
 const SingleContractData: React.FC<ISingleContractDataProps> = (props) => {
-  const { job } = useJob();
+  const { job, isLoading } = useJob();
   const { contract = job } = props;
 
-  return contract ? <DataTable contract={contract} /> : null;
+  return contract ? (
+    <DataTable contract={contract} loading={isLoading} />
```

Comment:
`isLoading` is always false here because job there will never be a case where `contract` is truthy and `isLoading` is also true.

I recommend removing the `contract` property from this component to avoid confusion.